vagrant/provisioning/roles/upgrade-solr/tasks/main.yml

```diff
@@ -1,14 +1,18 @@
 - name: check for available disk space
   become: yes
-  command: df -h "{{ root_folder }}/"
+  shell: df -h "{{ root_folder }}/" | awk 'NR==2 { print $4 }'
   register: space_available
 
 - name: check for solr data space
   become: yes
-  stat:
-    path:  "{{ root_folder }}/data/solr"
+  shell: du -sh "{{ root_folder }}/data/solr" | awk '{ print $1 }'
   register: solr_data
 
+- name: format free space and solr data size output
+  set_fact:
+    space_available_formated: "{{ space_available.stdout | human_to_bytes | int }}"
+    solr_data_formated: "{{ solr_data.stdout | human_to_bytes | int }}"
```

Comment:
This is from debugger, how the output will be look like for when conditions below.
TASK [upgrade-solr : debug space] ************************************************************************************************************************************************************
ok: [localhost] => {
    "msg": "102005473280"
}

TASK [upgrade-solr : debug solr] *************************************************************************************************************************************************************
ok: [localhost] => {
    "msg": "716800"
}